MapboxNavigation/RouteMapViewController.swift

```diff
@@ -137,6 +137,10 @@ class RouteMapViewController: UIViewController {
         mapView.camera = tiltedCamera
         mapView.setUserTrackingMode(.followWithCourse, animated: true)
         mapView.logoView.isHidden = false
+        
+        guard let controller = routePageViewController.currentManeuverPage else { return }
+        controller.step = currentStep
+        routePageViewController.notifyDidReRoute()
```

Comment:
Yeah... I agree, I hate a few of these functions that force updates to various parts of the UI.